### ISSUE TITLE
Use css classes

### ISSUE THREAD

**Issue by saharsaker** [lotus-1/LuckyPicture#14]
It is better to use css classes not the element in css. 
It is hard to use the html tags in css especially if your code will be longer later .

**Comment by Jasmineb2011** [comment]
Good point, the thing is our page is too short and based on js, not on styling.
